### ISSUE TITLE
website-service-worker.js breaks after redirecting.

### ISSUE THREAD

**Issue by foolip** [whatwg/whatwg.org#302]
After https://github.com/whatwg/whatwg.org/pull/301 + https://github.com/whatwg/misc-server/pull/126 , https://whatwg.org/mailing-list shows this error in Chrome for me:
> This site can’t be reached
> The webpage at https://whatwg.org/mailing-list might be temporarily down or it may have moved permanently to a new web address.

It works in incognito. The console (and @annevk) reveals that it's something broken with our SW setup:
```
website-service-worker.js:47 Could not refresh the cache for https://whatwg.org/mailing-list Error: https://whatwg.org/mailing-list is responding with 0
    at refreshCacheFromNetworkResponse (website-service-worker.js:58)
    at website-service-worker.js:45
```

What happened is that originally https://whatwg.org/mailing-list was a regular document and now it redirects to https://lists.whatwg.org/.

**Comment by foolip** [comment]
@domenic can you spot the problem in https://github.com/whatwg/whatwg.org/issues/302?

**Comment by domenic** [comment]
It appears redirects trigger neither of the guards in https://github.com/whatwg/whatwg.org/blob/c417dc3682bd0ca69ebfb783fc355d4819f077f1/resources.whatwg.org/website-service-worker.js#L57-L59. It looks like we should check for opaqueredirect as well. Maybe error too? https://fetch.spec.whatwg.org/#concept-response-type

However, that line which is giving the warning is just a warning saying that it can't refresh the cache. It's not the proximate cause of this issue, at least. So I'm still not sure whether this is the service worker's fault, or the HTTP cache.

**Comment by annevk** [comment]
Yeah, FWIW, in Firefox I end up at the cached version of `https://whatwg.org/mailing-list` and no errors.
